### ISSUE TITLE
Main menus cannot be closed via Esc

### ISSUE THREAD

**Issue by dozzzzer** [audacity/audacity#9687]
1. Open any top bar menu (File, Edit, etc.)
2. Press `Esc`
3. Observe: the menu won't close

**Comment by audio2000** [comment]
@dozzzzer
Hello again,
We both noticed the same issue again, thank you for closing my duplicate post I did not see this one when searching.
I put together a list of other items I noticed that do not close when using ESCAPE key (see below).
I was wondering, whenever you have some time, would you be able to help me confirm if any of the items I mentioned are duplicates, based on coding.  I am not sure if any of these items are the same when it comes to the coding used for them.
Thank You

Here is the link to the list if interested
#10071

**Comment by dozzzzer** [comment]
@saintmatthieu This is not a framework issue; MuseScore seems to handle closing via Esc gracefully.
